Description: A patch for autotrace.pc.in before 0.31.1-16
 This patch has been created to represent all changes was made direct
 to the package before implement dpkg-src 3.0(quilt) format, which has 
 a patch system.
 .
 Help to handle the transition to shlibs dependencies.
Author: Edgar Antonio Palma de la Cruz <xbytemx@gmail.com>
Last-Update: 2011-08-07

--- autotrace-0.31.1.orig/autotrace.pc.in
+++ autotrace-0.31.1/autotrace.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: Autotrace
 Description: a utility that converts bitmap to vector graphics
 Version: @VERSION@
-Requires: 
-Libs: -L@libdir@ -lautotrace @LIBPNG_LDFLAGS@ @MAGICK_LDFLAGS@ @LIBSWF_LDFLAGS@ @LIBPSTOEDIT_LIBS@
+Requires.private: @PKGCONFIG_REQS@
+Libs: -L@libdir@ -lautotrace
 Cflags: -I@includedir@
